src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -751,10 +767,22 @@ <h4 class="p2">Products that have been bought together</h4>
         </a>
       </template>
     </amp-list>
-    <!--
-    The button `tap` action sets the value of the `relatedProductsSrc` to an extended list of related products.
-    -->
-    <button [disabled]="listSrcButtonDisabled" class="ampstart-btn caps m1 mb3" id="listSrcButton" on="tap:AMP.setState({listSrcButtonDisabled: true, product: {relatedProductsSrc: '/json/more_related_products.json'}})">Show more</button>
+    <form method="GET"
+        action="/json/more_related_products"
+        action-xhr="/json/more_related_products"
+        target="_top"
+        on="submit-error: AMP.setState({
+          myState: { items: myState.items.concat(event.response.items)},
+          product: {  showMoreButtonDisabled: true }
+        }); submit-success: AMP.setState({
+          myState: { items: myState.items.concat(event.response.items)},
+          product: {timesShowMoreClicked: product.timesShowMoreClicked + 1 }
+        });">
+      <input type="hidden" name="timesShowMoreClicked" value="0" [value]="product.timesShowMoreClicked.toString()">
+      <input type="submit" value="Show more"
+        class="ampstart-btn caps m1 mb3 show"
+        [class] = "(product.showMoreButtonDisabled == true ? 'ampstart-btn caps m1 mb3 hide' : 'ampstart-btn caps m1 mb3 show')">
```

Comment:
Do you need `ampstart-btn caps m1 mb3 ` when the button is hidden?

Reply:
removed